modules/spring-data/src/main/java/org/apache/ignite/springdata/repository/support/IgniteRepositoryFactory.java

```diff
@@ -124,32 +121,29 @@ public IgniteRepositoryFactory(String springCfgPath) {
     }
 
     /** {@inheritDoc} */
-    @Override protected QueryLookupStrategy getQueryLookupStrategy(final QueryLookupStrategy.Key key,
-        EvaluationContextProvider evaluationCtxProvider) {
+    @Override protected Optional<QueryLookupStrategy> getQueryLookupStrategy(final QueryLookupStrategy.Key key,
+                                                                             EvaluationContextProvider evaluationCtxProvider) {
```

Comment:
Use 4 spaces instead of tab. And return second argument back to the left. Intendant should be 4 tabs, arguments shouldn't be aligned.